src/scripts/make_performance_tables.py

```diff
@@ -0,0 +1,299 @@
+            ("Overall Accuracy", ""),
+        ])
+    
+    # Round all numeric columns to 2 decimal places (excluding non-numeric columns like "Model")
+    numeric_columns = df.select_dtypes(include='number').columns
+    df[numeric_columns] = df[numeric_columns].applymap(lambda x: f"{x:.2f}".rstrip('0').rstrip('.'))
+
+
+   # Function to bold the best value (assuming higher is better)
+    def bold_best(series):
+        best = series.astype(float).max()  # Modify as per your criteria (e.g., use min() for smaller is better)
+        return series.apply(lambda x: f"\\textbf{{{x}}}" if float(x) == best else x)
+    
+    # Apply the bolding to relevant numeric columns (excluding 'Model')
+    if not human_subset:
+        columns_to_bold = [
+            ('Requires', 'Calculation'),
+            ('Requires', 'Knowledge'),
+            ('Requires', 'Reasoning'),
+            ('Requires', 'Intuition'),
+            ('Difficulty', 'Basic'),
+            ('Difficulty', 'Intermediate'),
+            ('Difficulty', 'Advanced'),
+            ('Overall Accuracy', '')
+        ]
+    else:
+        columns_to_bold = [
+            ('Requires', 'Calculation'),
+            ('Requires', 'Knowledge'),
+            ('Requires', 'Reasoning'),
+            ('Difficulty', 'Basic'),
+            ('Difficulty', 'Intermediate'),
+            ('Difficulty', 'Advanced'),
+            ('Overall Accuracy', '')
+        ]
+    
+    df[columns_to_bold] = df[columns_to_bold].apply(bold_best)
+
+    # Generate LaTeX table with bolded best values
+    latex_table = df.to_latex(index=False, multirow=True, escape=False)
+    
+    if not human_subset:
+        # Manually modify the LaTeX code for centering multi-level headers
+        latex_table = latex_table.replace(
+            r"Model & \multicolumn{4}{r}{Requires} & \multicolumn{3}{r}{Difficulty} & Overall Accuracy \\",
+            r"\multirow{3}{*}{Model} & \multicolumn{4}{c}{\textbf{Requires}} & \multicolumn{3}{c}{\textbf{Difficulty}} & \multirow{3}{*}{\textbf{Overall Accuracy}}\\"
+            r"\cmidrule(lr){2-5} \cmidrule(lr){6-8}\\"
+        )
+        latex_table = latex_table.replace(
+            r"\begin{tabular}{lllllllll}",
+            r"\begin{tabular}{ccccccccc}"
+        )
+    else:
+        # Manually modify the LaTeX code for centering multi-level headers
+        latex_table = latex_table.replace(
+            r"Model & \multicolumn{3}{r}{Requires} & \multicolumn{3}{r}{Difficulty} & Overall Accuracy \\",
+            r"\multirow{3}{*}{Model} & \multicolumn{3}{c}{\textbf{Requires}} & \multicolumn{3}{c}{\textbf{Difficulty}} & \multirow{3}{*}{\textbf{Overall Accuracy}}\\"
+            r"\cmidrule(lr){2-4} \cmidrule(lr){5-7}\\"
+        )
+        latex_table = latex_table.replace(
+            r"\begin{tabular}{llllllll}",
+            r"\begin{tabular}{cccccccc}"
+        )
+
+    # Wrap the table with \resizebox
+    latex_table = (
+        r"\begin{table}[htbp]\centering" + "\n" +
```

Comment:
I think I'd only put the `latex_table` in the file and then handle caption & co in the latex file